CONTRIBUTING.md

```diff
@@ -1,126 +1,68 @@
-> The tests will be skipped when using `-Dquickly` without `-DskipTests=false`.
-
-#### Test Troubleshooting
-
-- If you encounter issues (like `java.lang.UnsatisfiedLinkError: failed to load the required native library`) while running the test StandaloneGatewaySecurityTest.shouldStartWithTlsEnabled take a look at https://github.com/camunda/camunda/issues/10488 to resolve it.
+# Contributing to Camunda
+
+We welcome new contributions. We take pride in maintaining and encouraging a friendly, welcoming, and collaborative community.
+
+Anyone is welcome to contribute to Camunda! The best way to get started is to choose an existing [issue](#starting-on-an-issue).
+
+Core code contributions must fit into the scope of Camunda products. The ultimate decision is up to the core Camunda team. If your vision does not fit in the core Camunda product, you may be directed to the [Camunda Community Hub](https://github.com/camunda-community-hub) and/or [Camunda Marketplace](https://marketplace.camunda.com/en-US/home) instead.
+
+- [Prerequisites](#prerequisites)
+  - [Contributor License Agreement](#contributor-license-agreement)
+  - [Code of Conduct](#code-of-conduct)
+- [GitHub issue guidelines](#github-issue-guidelines)
+  - [Starting on an issue](#starting-on-an-issue)
+- [Creating a pull request](#creating-a-pull-request)
+- [Reviewing a pull request](#reviewing-a-pull-request)
+  - [Review emoji code](#review-emoji-code)
+  - [Stale pull requests](#stale-pull-requests)
+- [Backporting changes](#backporting-changes)
+- [Commit message guidelines](#commit-message-guidelines)
+  - [Commit message header](#commit-message-header)
+  - [Commit message body](#commit-message-body)
+- [Build Camunda from source](#build-camunda-from-source)
+  - [Test Execution](#test-execution)
+    - [Test Troubleshooting](#test-troubleshooting)
+  - [Build profiling](#build-profiling)
+
+## Prerequisites
+### Contributor License Agreement
+
+You will be asked to sign our [Contributor License Agreement](https://cla-assistant.io/camunda-community-hub/community) when you open a Pull Request. We are not
+asking you to assign copyright to us but to give us the right to distribute
+your code without restriction. We ask this of all contributors to
+assure our users of the origin and continuing existence of the code. 
 
-### Build profiling
+*Note: In most cases, you will only need to sign the CLA once.* 
 
-The development team continues to push for a performant build.
-To investigate where the time is spent, you can run your maven command with the `-Dprofile` option.
-This will generate a profiler report in the `target` folder.
+### Code of Conduct
 
-## Report issues or contact developers
+This project adheres to the [Camunda Code of Conduct](https://camunda.com/events/code-conduct/).
+By participating, you are expected to uphold this code. Please [report](https://camunda.com/events/code-conduct/reporting-violations/)
+unacceptable behavior as soon as possible.
```

Comment:
🔧 I recommend wrapping lines (i.e. starting new lines) for new sentences only. This nudges authors to rewrite sentences to be shorter which typically leads to simpler text that is easier to read. It also makes it easier to review change in git because it highlights changes by removing and adding individual lines.

Reply:
I think I fixed the text wrapping in all the spots where it was, but let me know if you spot any other places.